app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginViewModel.java

```diff
@@ -76,8 +92,11 @@ public void setBaseUrl(String url, boolean shouldSetDefaultUrl) {
         interceptor.setInterceptor(baseUrl);
     }
 
-    public Login getLogin() {
-        return login;
+    public LiveData<Login> getLogin() {
+        login.setEmail(encryptionService.decrypt(sharedPreferenceModel.getString(PREF_USER_EMAIL, null)));
+        login.setPassword(encryptionService.decrypt(sharedPreferenceModel.getString(PREF_USER_PASSWORD, null)));
+        decryptedLogin.setValue(login);
```

Comment:
Only do this once

Reply:
The setting of email, password, decrypting of email, password, and setting of decrypted login

Reply:
I am already doing it once.

Reply:
Every time getLogin is called, this happens

Should only happen once